explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -291,16 +291,12 @@ object ObsTabTiles:
           // asterism base coordinates at viz time or default to base coordinates
```

Comment:
Do we want to default this to the base coordinates or to the current time?

Reply:
For one thing, when we add non-sidereals, we'll lose the concept of base coordinates.

Reply:
So we'd likke to have reference coordinates that are not time dependent
That probably make no sense for a non-sidereals though we could have mixed asetrisms right?

Reply:
Right. But it works for now. Non-sidereals are going to require a lot of changes...

Reply:
Yeah it will be a big task